en/docs/reference/connectors/cdc-inbound-endpoint/cdc-inbound-endpoint-example.md

```diff
@@ -0,0 +1,154 @@
+# CDC Inbound Endpoint Example 
+
+The CDC (Change Data Capture) inbound endpoint enables you to capture changes made to data in a database. Consider a scenario where the database crashes at some point, leading to the loss of data related to certain operations. This highlights the need to synchronize the data with a cloud-based backup storage. To minimize risk, updating the cloud storage in real time is preferable to batch updates. Therefore, it is important to detect modification events and update the cloud storage accordingly. The CDC inbound endpoint can be used for this purpose.
+
+## What you'll build
+
+This scenario demonstrates how the CDC inbound endpoint can be used to capture changes occurring in a MySQL database. In this example, we will configure the **CDC Inbound Endpoint** to capture changes made to a **MySQL** table, such as insertions, updates, and deletions.
+
+In this example, a relational database table is used to store product information. The product data is added to the database by an external system that is outside of the enterprise's control. As soon as a new product is inserted, the system needs to detect and process the data. The integration runtime is used here to listen for database changes and trigger the relevant processes. It can either invoke backend APIs or place the data onto a message bus after performing the necessary data transformations. However, for the sake of simplicity in this example, we will simply log the message. You can extend this example as needed using WSO2 mediators.
+
+## Set up the environment 
+
+1. First, install [MySQL database](https://www.mysql.com/) locally. If you have a remote server, please obtain the credentials required to connect.  
+
+2. Connect to MySQL server.
+
+3. Create a database called `inventory`. 
+    ```sql
+    CREATE DATABASE inventory;
+    ```
+
+4. Then create a table called `products` under that database using the following SQL script. 
+    ```sql
+    CREATE TABLE products (
+        id INT AUTO_INCREMENT PRIMARY KEY,
+        name VARCHAR(255) NOT NULL,
+        price REAL NOT NULL
+    );
+    ```
+
+6. Download the latest Debezium Orbit JAR for **2.1.4.Final.wso2v2** from [nexus](https://maven.wso2.org/nexus/content/repositories/public/org/wso2/orbit/debezium/debezium/) and place it in `<Project Home>/deployment/libs` directory.
```

Comment:
While the text says to download ` 2.1.4.Final.wso2v2` the given link has only `2.1.4.Final.wso2v1`

Reply:
It is now available in the given link.

Reply:
Shall we avoid the jar name and just say to download the latest Debezium Orbit JAR? This way, we don't have to change the doc every time we do a JAR release.

Reply:
Yes That is better

Reply:
I fixed it.